src/environments/environment.ts

```diff
@@ -1,9 +1,11 @@
 // This file can be replaced during build by using the `fileReplacements` array.
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
+// isDebugMode from 0 to 3 when 0 then logger completely disabled
 
 export const environment = {
-  production: false
+  production: false,
+  isDebugMode: 3
```

Comment:
loggingLevel would be a better name for this.  isXXX implies a boolean.

Reply:
Also, is there a way of maintaining a map for a meaningful level to map what you showed yesterday:

1=ERROR
2=WARN
3=INFO
4=DEBUG

so we could set it like this:

```
loggingLevel: 'WARN'
```

or 

```
loggingLevel: Level.WARN
```